install_octopus.sh

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash
+# This script prepares the download of octopus in the right location given the version number, location to untar / clone and install prefix
+# example run:
+# $ ./install_octopus.sh 13.0 /opt/octopus /home/user/octopus-bin
+# $ ./install_octopus.sh develop /opt/octopus
+
+# exit on error and print each command
+set -xe
+
+# Check if the version number and location is provided
+if [ -z "$1" ]
+  then
+    echo "No version number provided"
+    exit 1
+else
+  version=$1
+fi
+if [ -z "$2" ]
+  then
+    echo "No download location provided"
+    exit 1
+else
+    location=$2
+fi
+if [ -z "$3" ]
+  then
+    echo "No install prefix provided using default location"
+    prefix=/usr/local
+else
+    prefix=$3
+fi
+
+# Convenience tools (up to emacs)
```

Comment:
I would move all the `apt-get` commands into a separate bash script:
- people not using debian can decide not to run this
- somebody on debian may not want to run `apt-get` but rather see if the compilation of octopus works with the already installed packages

Reply:
Im not too sure, the configure line depends on what you have installed and where, thus the two are tied together.
Maybe we could do that refactor if any user requests something like that later.

So far the script is synonymous to " one click octopus install" , so primarily targeted for users who would not want to tinker into the install scripts but rather just want to get octopus running. I would say given the primary idea of this repo ( to provide a docker image with octopus) personalised scripts would likely not fit in here.

Thoughts?

Reply:
You are right that this is somewhat specific to Debian, and that the change from one script to two would make this a little bit less convenient.

However, so far, the script must be run a root (or sudo). If we split it up, also a 'normal' user can do it. I think that is important: if I am not in a container, I would not want to run random `apt-get` install commands (and would also not want to be root to compile octopus) on my linux system.

Reply:
That is a compelling argument, installing packages need root permissions and should thus be done in isolation to compiling the code. I have updated the MR to reflect this